### PR TITLE
Add miniWOL.app v2.1.3

### PR DESCRIPTION
**Pull request by gzurowski**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/miniwol.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 "3a42570e704616620a74969ce180151553751bbcd33e34cd02194ac4edf50b3a"
+
+  url "https://www.tweaking4all.com/downloads/network/miniWOL-v#{version}-64bit.dmg"
+  name "miniWOL"
+  desc "Small menu bar tool for sending Wake on LAN (WOL) network packets"
+  homepage "https://www.tweaking4all.com/network-internet/miniwol2/"
+
+  livecheck do
+    url "https://www.tweaking4all.com/downloads/"
+    regex(/href=.*?miniWOL-v?(\d+(?:\.\d+)+)-64bit\.dmg/i)
```
Comment (bevanjkay) [inline Casks/miniwol.rb:12]:
```suggestion
    regex(/href=.*?miniWOL[._-]v?(\d+(?:\.\d+)+)-64bit\.dmg/i)
```
Small adjustments to follow standards.

**Review by bevanjkay — DISMISSED** [review]
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Review by SMillerDev — APPROVED** [review]

**Comment by gzurowski** [comment]
> Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

I've implemented the requested changes, please take a look @bevanjkay.

**Comment by miccal** [comment]
Thank you @gzurowski.

**Pull request merged**